### PR TITLE
feat(css): lowercase html tags

### PR DESCRIPTION
**Pull request by evilebottnawi**
All html tag in lowercase format. Also lowercase `from/to` for keyframes.

### CODE REVIEW

File: tests/css_case/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -137,7 +218,7 @@ $KeepScssVar: val;
 
 @import Keep;
 
-HTML#KeepId.KeepClass,
+html#KeepId.KeepClass,
 a[HREF="KeepAttrValue"]:hover::first-letter,
 svg[viewBox] linearGradient,
```
Comment (evilebottnawi) [inline tests/css_case/__snapshots__/jsfmt.spec.js.snap]: @j-f1 no, we have test on this

Comment (evilebottnawi) [reply tests/css_case/__snapshots__/jsfmt.spec.js.snap]: only html tags

**Comment by evilebottnawi** [comment]
@j-f1 done :+1:

**Comment by evilebottnawi** [comment]
@j-f1 seems you did not write a comment :smile:

**Comment by j-f1** [comment]
@evilebottnawi I dismissed my review because the merge conflicts were fixed.

**Comment by evilebottnawi** [comment]
@lydell @j-f1 approve?

**Pull request merged**